docs/index.asciidoc

```diff
@@ -57,7 +57,12 @@ output plugins.
   * Value type is <<number,number>>
   * Default value is `1420`
 
-The GELF chunksize. You usually don't need to change this.
+The 
```

Comment:
what was the reason for this whitespace? I'm guessing a mistake :)

Reply:
indeed ! :)